api/Element.json

```diff
@@ -5416,7 +5416,7 @@
               "version_added": "31"
             },
             "chrome_android": {
-              "version_added": null
+              "version_added": "31"
```

Comment:
Can the mousewheel event actually be fired on Chrome for Android? I imagine this has to depend on mouse support in Android, and that didn't always exist?

Reply:
I'm not sure exactly when Android supported a mouse, but I found this article that [talks about their mouse support in Android 4.0](https://kelbin.medium.com/mouse-support-in-android-b87f0c95b78d).  Considering the time when Android 4.0 was released (Q4 2011) and the time Chrome 31 was released (Q4 2013), this seems reasonable that the event would be supported at the same time as it was implemented in Chrome.

Reply:
Good digging, that makes this very plausible indeed.